.travis.yml

```diff
@@ -14,20 +14,29 @@ jobs:
       services:
         - xvfb
       before_install:
-        - mkdir qunit
-        - wget -O qunit/qunit-1.18.0.css http://code.jquery.com/qunit/qunit-1.18.0.css
-        - wget -O qunit/qunit-1.18.0.js  http://code.jquery.com/qunit/qunit-1.18.0.js
+        - sudo add-apt-repository -y ppa:deadsnakes/ppa
+        - sudo apt-get -q update
+        - sudo apt-get -y install python3.10
```

Comment:
Why do all this?  Travis CI Jammy already has Py3.10.5 builtin
https://docs.travis-ci.com/user/reference/jammy/#python-support

Reply:
Great! I didn't realize that.